src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -237,6 +239,22 @@ void ThreadStateBar::DoUpdatePrimitives(PrimitiveAssembler& primitive_assembler,
         const Vec2 pos{x0, GetPos()[1]};
         const Vec2 size{width, GetHeight()};
 
+        if (slice == app_->selected_thread_state_slice()) {
+          const Color kWhiteBorder = Color(255, 255, 255, 255);
```

Comment:
Nit: kBorderColor (Having the color itself in the name defies the purpose of the constant a bit)